content/blog/2019/announcing-1.1/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Announcing Istio 1.1
+description: Istio 1.1 release announcement.
+publishdate: 2019-03-14
+attribution: The Istio Team
+weight: 77
```

Comment:
Please remove weight: here, we don't need this anymore.

Reply:
ack fixed